src/hotspot/share/nmt/memReporter.cpp

```diff
@@ -181,14 +182,14 @@ void MemSummaryReporter::report() {
     MemTag mem_tag = NMTUtil::index_to_tag(index);
     // thread stack is reported as part of thread category
     if (mem_tag == mtThreadStack) continue;
-    MallocMemory* malloc_memory = _malloc_snapshot->by_type(mem_tag);
-    VirtualMemory* virtual_memory = _vm_snapshot->by_type(mem_tag);
+    MallocMemory* malloc_memory = _malloc_snapshot->by_tag(mem_tag);
+    VirtualMemory* virtual_memory = _vm_snapshot->by_tag(mem_tag);
 
-    report_summary_of_type(mem_tag, malloc_memory, virtual_memory);
+    report_summary_of_tag(mem_tag, malloc_memory, virtual_memory);
   }
 }
 
-void MemSummaryReporter::report_summary_of_type(MemTag mem_tag,
+void MemSummaryReporter::report_summary_of_tag(MemTag mem_tag,
```

Comment:
Move out into mainline PR

Reply:
Fixed in commit "missed flag/tag -> type" (7640108).